bin/check-marathon-apps.rb

```diff
@@ -311,8 +335,11 @@ def post_check_result(data)
                     data.to_json,
                     content_type: 'application/json',
                     timeout: config[:timeout])
+    true
   rescue RestClient::ExceptionWithResponse => e
-    critical "Error while trying to POST check result (#{config[:sensu_client_url]}/results): #{e.response}"
+    # print a message about failing POST but keep going
+    STDERR.puts "Error while trying to POST check result for #{data} (#{config[:sensu_client_url]}/results): #{e.response}"
+    false
```

Comment:
:+1: it has always bothered me every time I see a function other than run that uses `(critical|warning|unknown)` rather than returning false as it makes it harder to write unit tests.